src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -179,12 +178,19 @@ private void applyStringFilter(String filterString) {
             } else {
                 this.applyFilterExpression(Qualifier.EMPTY);
             }
-        } catch (ParseException ex) {
+        } catch (FilterException ex) {
             this.applyFilterExpression(Qualifier.EMPTY);
             // Overrides message in status bar
-            UI.status.displayMessage("Panel " + (panelIndex + 1)
-                + ": Parse error in filter: " + ex.getMessage());
+            UI.status.displayMessage(resolvePanelInfo(panelMenuBar.getPanelName())
+                + ": " + ex.getMessage());
+        }
+    }
+
+    private String resolvePanelInfo(String panelName) {
+        if (panelName.equals("Panel")) {
```

Comment:
Should I add the a string constant to [PanelMenuBar.java](https://github.com/HubTurbo/HubTurbo/blob/master/src/main/java/ui/components/PanelMenuBar.java#L43) where the panel name is initialized ?